recipe/meta.yaml

```diff
@@ -25,18 +27,22 @@ requirements:
 test:
   imports:
     - chardet
+  requires:
+    - pip
   commands:
+    - pip check
     - chardetect --help
 
 about:
   home: https://github.com/chardet/chardet
-  license: LGPL2
+  license: LGPL-2.1-only 
   license_family: GPL
   license_file: LICENSE
+  license_url: https://github.com/chardet/chardet/blob/main/LICENSE
```

Comment:
as package includes lincese_file, the tag license_url is superflous

Reply:
same with this, I'll go ahead and remove it.